bundles/org.openhab.binding.radiothermostat/src/main/java/org/openhab/binding/radiothermostat/internal/handler/RadioThermostatHandler.java

```diff
@@ -119,7 +118,7 @@ public void initialize() {
         this.disableLogs = config.disableLogs;
         this.clockSync = config.clockSync;
 
-        if (hostName == null || "".equals(hostName)) {
+        if (hostName == null || BLANK.equals(hostName)) {
```

Comment:
```suggestion
        if (hostName == null || hostName.isBlank()) {
```

Reply:
fixed

Reply:
I would propose to accept my suggestion so that a string of only whitespaces would not be accepted either.

Reply:
ok, got it now.